pom.xml

```diff
@@ -75,14 +75,14 @@
 	<properties>
 		<!-- General -->
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<java.version>11</java.version>
+		<java.version>8</java.version>
 
 		<!-- Deps version -->
 		<powermock.version>2.0.9</powermock.version>
 		<junit.version>4.13.2</junit.version>
 		<jmh.version>1.19</jmh.version>
 		<neo4j.java.driver.version>4.3.1</neo4j.java.driver.version>
-		<neo4j.version>4.0.0</neo4j.version>
+		<neo4j.version>4.3.0</neo4j.version>
```

Comment:
do we want to use to parameterize the Docker image coordinates? (if so, define a GH Actions matrix with 3.5, 4.0, 4.1, 4.2 and 4.3 as well?)

Reply:
Maybe we can do this in a different PR? WDYT?

Reply:
Sounds good to me